src/featbin/extract-feature-segments.cc

```diff
@@ -45,6 +45,8 @@ int main(int argc, char *argv[]) {
     BaseFloat min_segment_length = 0.1,  // Minimum segment length in seconds.
         max_overshoot = 0.0;  // max time by which last segment can overshoot
     BaseFloat samp_freq = 100;  // feature sampling frequency (assuming 10ms window shift)
```

Comment:
This is an 'orphan' program, it is used nowhere in the scripts.  Therefore we can make non-back-compatible changes.
The samp-freq option should be changed to --frame-shift -- obviously this changes its meaning -- and your --trim-starts option should be changed to --snip-edges, both to match with things like compute-mfcc-feats.  (note: using --snip-edges=false when dumping features will make your life easier, but we can support the option here too).  And try to make their documentation compatible with that in compute-mfcc-feats.